src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -9,12 +9,19 @@
   public WasapiDownloaderSettings settings = null;
 
 
-  public WasapiDownloader(String settingsFileLocation, String[] args) throws IOException {
+  public WasapiDownloader(String settingsFileLocation, String[] args) throws LoadSettingsException {
     settings = new WasapiDownloaderSettings(settingsFileLocation, args);
   }
 
+  public void run() {
+    if(settings.shouldDisplayHelp()) {
```

Comment:
i slightly prefer no space, and have done that more over the years when writing java, but i changed it to have a space, since that's what we do in the wowza codebase, and that's what we started doing in this one (didn't feel strongly enough about it to break consistency with prior code).